### PR TITLE
feat: adds AcrossFacetPacked (v1.0.0) incl. deploy/update scripts

### PR DESCRIPTION
**Pull request by 0xDEnYO**

### CODE REVIEW

File: src/Facets/AcrossFacetPacked.sol [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+            BridgeData memory bridgeData,
+            AcrossFacet.AcrossData memory acrossData
+        )
+    {
+        require(data.length >= 80, "invalid calldata (must have length > 80)");
+
+        // extract bridgeData
+        bridgeData.transactionId = bytes32(bytes8(data[4:12]));
+        bridgeData.receiver = address(bytes20(data[12:32]));
+        bridgeData.destinationChainId = uint64(uint32(bytes4(data[32:36])));
```
Comment (maxklenk) [inline src/Facets/AcrossFacetPacked.sol:261]: for consistency here as well. I guess currently it is casted automatically as the `bridgeData` interface uses `uint256`

**Review by maxklenk — CHANGES_REQUESTED** [review]
Looks good, please create a staging deployment, set the approvals and test it with the backend.

File: src/Facets/AcrossFacetPacked.sol [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+                message
+            );
+    }
+
+    function encode_startBridgeTokensViaAcrossERC20Packed(
+        bytes32 transactionId,
+        address receiver,
+        address sendingAssetId,
+        uint256 minAmount,
+        uint64 destinationChainId,
```
[the remark this answers is not in the document]
Comment (0xDEnYO) [reply src/Facets/AcrossFacetPacked.sol:211]: updated

File: src/Facets/AcrossFacetPacked.sol (hunk 1 above)
Comment (0xDEnYO) [reply src/Facets/AcrossFacetPacked.sol:261]: yes correct, it's casted automatically

**Review by maxklenk — APPROVED** [review]

**Review by ezynda3 — APPROVED** [review]

**Pull request merged**